.github/actions/clean_packages/clean_packages.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2023 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+
+import os
+
+import requests
+from rich import print
+
+""" Use current GitHub API to check to llst packages
+    in registry and remove all but last 3 or custom
+    set number of packages.
+    Reference: https://docs.github.com/en/rest/packages/packages?apiVersion=2022-11-28#about-github-packages
+"""
+
+dry_run: bool = True if os.getenv("INPUT_DRY_RUN") == "true" else False
+keep = int(os.getenv("INPUT_KEEP"))
+org = os.getenv("GITHUB_REPOSITORY_OWNER")
+packages = os.getenv("INPUT_PACKAGES").split("\n")
+token = os.getenv("INPUT_TOKEN")
+
+print(packages)
+
+headers = {
+    "Accept": "application/vnd.github+json",
+    "Authorization": f"Bearer {token}",
+    "X-GitHub-Api-Version": "2022-11-28",
+}
+
+# Assembly organization packages url string
+pkg_url: str = f"https://api.github.com/orgs/{org}/packages"
+
+
+def remove_packages():
```

Comment:
Suggest to rename to "delete_packages" to use consistent name with function description and in the GitHub API delete is used

Reply:
done